pkg/registry/core/namespace/strategy.go

```diff
@@ -47,6 +48,17 @@ func (namespaceStrategy) NamespaceScoped() bool {
 	return false
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (namespaceStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	return map[fieldpath.APIVersion]*fieldpath.Set{
+		"v1": fieldpath.NewSet(
+			fieldpath.MakePathOrDie("status"),
+			fieldpath.MakePathOrDie("spec.finalizers"),
```

Comment:
Does this actually work with that "." in the middle?

Reply:
:-( You know it doesn't

Reply:
(multiple occurrences)